### PR TITLE
Add description of multiscales metadata 0.2

### PR DESCRIPTION
**Pull request by constantinpape**
This was part of #39

### CODE REVIEW

File: 0.2/index.bs [hunk 1]
```diff
@@ -207,8 +207,48 @@ keys as specified below for discovering certain types of data, especially images
...
+    "multiscales": [
+        {
+            "version": "0.2",
+            "name": "example",
+            "datasets": [
+                {"path": "0"},
+                {"path": "1"},
+                {"path": "2"}
+            ],
+            "axes": [
```
Comment (joshmoore) [inline 0.2/index.bs:237]: Want to drop these three lines?

File: 0.2/index.bs [hunk 2]
```diff
@@ -207,8 +207,48 @@ keys as specified below for discovering certain types of data, especially images
 
 Metadata about the multiple resolution representations of the image can be
 found under the "multiscales" key in the group-level metadata.
-The specification for the multiscale (i.e. "resolution") metadata is provided
-in [zarr-specs#50](https://github.com/zarr-developers/zarr-specs/issues/50).
+
+"multiscales" contains a list of dictionaries where each entry describes a multiscale image.
+
+Each "multiscales" dictionary MUST contain the field "datasets", which is a list of dictionaries describing
```
Comment (joshmoore) [inline 0.2/index.bs:213]:
```suggestion
Each dictionary contained in the list MUST contain the field "datasets", which is a list of dictionaries describing
```
or "Each multiscale dictionary..."?

File: 0.2/index.bs (hunk 1 above)
Comment (constantinpape) [reply 0.2/index.bs:237]: Good point, I missed that in the example...

Comment (constantinpape) [reply 0.2/index.bs]: Done.

File: 0.2/index.bs [hunk 3]
```diff
@@ -207,8 +207,48 @@ keys as specified below for discovering certain types of data, especially images
...
+Each dictionary in "datasets" MUST contain the field "path", whose value contains the path to the array for this resolution relative
+to the current zarr group. The "path"s MUST be ordered from largest (i.e. highest resolution) to smallest.
+
+It SHOULD contain the field "name".
+
+It SHOULD contain the field "version", which indicates the version of the
+multiscale metadata of this image (current version is 0.2).
+
+It SHOULD contain the field "type", which gives the type of downscaling method used to generate the multiscale image pyramid.
+It SHOULD contain the field "metadata", which contains a dictionary with additional information about the downscaling method.
```
Comment (joshmoore) [inline 0.2/index.bs:224]: Add a newline. cf. http://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/constantinpape/ngff/4fff9dafb41f2282c69afc1f852fc1d5ce117569/0.2/index.bs

**Review by joshmoore — COMMENTED** [review]
Few minor comments, but generally huge thanks!

**Comment by joshmoore** [comment]
Thanks, @constantinpape :+1:

**Pull request merged**